### PR TITLE
libeemd 1.4 (new formula)

### PR DESCRIPTION
**Pull request by keiichishima**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x] Written a sensible test? (The best test is to compile and run a sample program.)

libeemd is a C library for performing the ensemble empirical mode
decomposition.

### CODE REVIEW

File: libeemd.rb [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  homepage "https://bitbucket.org/luukko/libeemd"
+  url "https://bitbucket.org/luukko/libeemd/get/v1.4.tar.gz"
+  sha256 "c484f4287f4469f3ac100cf4ecead8fd24bf43854efa63650934dd698d6b298b"
+
+  depends_on "gsl"
+  depends_on "pkg-config" => :build
+
+  needs :openmp
+
+  patch :DATA
```
Comment (jonchang) [inline libeemd.rb]: Please add a comment explaining what the patch does and if it has been reported upstream (or why it hasn't been).

File: libeemd.rb [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
++	install libeemd.$(version).dylib $(PREFIX)/lib
++	cp -Pf libeemd.dylib $(PREFIX)/lib
+ 
+ uninstall:
+ 	@echo "$$uninstall_msg"
+@@ -44,14 +44,14 @@
+ 	mkdir -p obj
+ 
+ obj/eemd.o: src/eemd.c src/eemd.h | obj
+-	gcc $(commonflags) -c $< $(gsl_flags) -o $@
```
Comment (jonchang) [inline libeemd.rb]: Replacing `gcc` with `$(CC)` shouldn't be necessary as Homebrew ought to automatically pick the correct compiler regardless of what you call it. Is this required for this formula to build?

File: libeemd.rb [hunk 3]
```diff
@@ -0,0 +1,102 @@
+class Libeemd < Formula
+  desc "Library for performing the ensemble empirical mode decomposition"
+  homepage "https://bitbucket.org/luukko/libeemd"
+  url "https://bitbucket.org/luukko/libeemd/get/v1.4.tar.gz"
+  sha256 "c484f4287f4469f3ac100cf4ecead8fd24bf43854efa63650934dd698d6b298b"
```
Comment (jonchang) [inline libeemd.rb]: Consider adding a `head` bitbucket URL and a `doi` tag like in [raxml.rb](https://github.com/Homebrew/homebrew-science/blob/master/raxml.rb)

**Comment by keiichishima** [comment]
Thank you for the review.  I've updated the code and pushed to the repository.  One thing I'm worried about is about the local patch.  As described in the code comment, I'm not going to request an update to upstream because it requires a big build system change to the original code while we don't get many benefits from it...

**Comment by jonchang** [comment]
Thanks @keiichishima! This has been merged.

**Pull request closed without merging**